app/views/devise/shared/_registration_form.html.slim

```diff
@@ -1,54 +1,53 @@
 = form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
-  form.form-horizontal[novalidate]
-    .row
-      = devise_error_messages!
-      .col-12.col-sm-6.col-md-6
-        fieldset.form-group.position-relative.has-icon-left
-          - f.label :first_name
-          = f.text_field :first_name,
+  .row
+    = devise_error_messages!
+    .col-12.col-sm-6.col-md-6
+      fieldset.form-group.position-relative.has-icon-left
+        - f.label :first_name
+        = f.text_field :first_name,
                   class: "form-control input-lg",
                   placeholder:"First Name",
                   autofocus: true
-          .form-control-position
-            i.fa.fa-user
-      .col-12.col-sm-6.col-md-6
-        fieldset.form-group.position-relative.has-icon-left
-          - f.label :last_name
-          = f.text_field :last_name,
+        .form-control-position
+          i.fa.fa-user
+    .col-12.col-sm-6.col-md-6
+      fieldset.form-group.position-relative.has-icon-left
+        - f.label :last_name
```

Comment:
and more

Reply:
@Misha7776 I believe Roman wonders why did you place ```-``` instead of ```=``` at the beginning of the line.

Reply:
i placed - instead of = because didn`t want to output the labels